<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesWidget</class>
 <widget class="QWidget" name="PropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedProperties">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupTorrentInfos">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="title">
            <string>Torrent information</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="savePath_lbl">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Save path:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="creator_lbl">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Creator:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="hash_lbl2">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Torrent hash:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="comment_lbl2">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Comment:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="save_path">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="changeSavePathButton">
                  <property name="maximumSize">
                   <size>
                    <width>27</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="creator">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="hash_lbl">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="comment_lbl">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>185</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>115</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="title">
            <string>Current session</string>
           </property>
           <layout class="QVBoxLayout" name="_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>9</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="_3">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="_4">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>Sans Serif</family>
                    <pointsize>9</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="text">
                   <string>Total uploaded:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <family>Sans Serif</family>
                    <pointsize>9</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="text">
                   <string>Total downloaded:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lbl_ratio">
                  <property name="font">
                   <font>
                    <family>Sans Serif</family>
                    <pointsize>9</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="text">
                   <string>Share ratio:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="font">
                   <font>
                    <family>Sans Serif</family>
                    <pointsize>9</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="text">
                   <string>Total failed:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="_5">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="upTotal">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="dlTotal">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="shareRatio">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="failed">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>181</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="RealProgressBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Downloaded pieces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="incrementalDownload">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Download in correct order (slower but good for previewing)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="main_infos_button">
       <property name="text">
        <string>General</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/oxygen/help-about.png</normaloff>:/Icons/oxygen/help-about.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trackers_button">
       <property name="text">
        <string>Trackers</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/oxygen/network-server.png</normaloff>:/Icons/oxygen/network-server.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="url_seeds_button">
       <property name="text">
        <string>URL seeds</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/oxygen/urlseed.png</normaloff>:/Icons/oxygen/urlseed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="files_button">
       <property name="text">
        <string>Files</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/oxygen/folder.png</normaloff>:/Icons/oxygen/folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
